core-services/sanitize-prow-jobs/_config.yaml

```diff
@@ -3838,8 +3838,8 @@ groups:
     - periodic-ci-openshift-cluster-autoscaler-operator-e2e-azure-master
     - periodic-ci-openshift-cluster-autoscaler-operator-e2e-gcp-master
     - periodic-openshift-library-import
-    - release-openshift-origin-installer-e2e-remote-libvirt-ppc64le-4.7
-    - release-openshift-origin-installer-e2e-remote-libvirt-s390x-4.7
+    - release-openshift-ocp-installer-e2e-remote-libvirt-ppc64le-4.7
```

Comment:
This is the missing step in the card.
We can remove those job names in the config file.
They are controlled by `sshBastion: build01`, which assigns all the jobs with `dptp.openshift.io/ssh-bastion: "true"` label the `build01` regardless of what the rest of the config file looks like.